Dockerfile.fortran

```diff
@@ -0,0 +1,70 @@
+# versions and sizes from here: https://hub.docker.com/r/intel/oneapi-hpckit/tags
+FROM intel/oneapi-hpckit:latest
+
+RUN apt update \
+    && apt -y install \
+    cmake \
+    cmake-curses-gui \
+    curl \
+    libcurl4-openssl-dev \
+    libhdf5-dev \
+    m4 \
+    vim \
+    zlib1g-dev \
+    git \
+    lcov \
+    make \
+    libnetcdff-dev \
+    valgrind \ 
+    gcc \
+    gfortran \
+    && apt clean 
+
+# Set environment variables to install MUSICA using gcc
+ENV FC=gfortran
+ENV FFLAGS="-I/usr/include/"
+
+# Install json-fortran for gnu version
+RUN curl -LO https://github.com/jacobwilliams/json-fortran/archive/8.2.0.tar.gz \
+    && tar -zxvf 8.2.0.tar.gz \
+    && cd json-fortran-8.2.0 \
+    && mkdir build \
+    && cd build \
+    && cmake -D SKIP_DOC_GEN:BOOL=TRUE .. \
+    && make install
+
+# Copy the musica code
+COPY . musica
+
+# Set json-fortran variable to install MUSICA using gcc
+ENV JSON_FORTRAN_HOME="/usr/local/jsonfortran-gnu-8.2.0"
+
+# Build and install MUSICA
+RUN cd musica \
+    && cmake -S . \
+             -B build \
+             -D ENABLE_TESTS=ON \
+    && cd build \
+    && make install -j 8 
+
+# Set environment variables to build MUSICA-Fortran using intel compilers
+ENV CC=icx
+ENV CXX=icpx
+ENV FC=ifort
+
+# Install json-fortran for intel version 
+RUN cd json-fortran-8.2.0 \
+    && mkdir build-intel \
+    && cd build-intel \
+    && cmake -D SKIP_DOC_GEN:BOOL=TRUE .. \
+    && make install
+
+# Set json-fortran variable to build MUSICA-Fortran using intel
+ENV JSON_FORTRAN_HOME="/usr/local/jsonfortran-intel-8.2.0"
+
+RUN cd musica/musica-fortran/test \
+    && mkdir build && cd build \
+    && cmake .. \
```

Comment:
Does this not need the `USE_MUSICA_FORTRAN=ON`? Or maybe I'm not following the logic correctly

Reply:
It's an entirely new cmake project. Internally it uses fetch content to pull musica and in that cmake file it sets use_musica_fortran=on